maintain/going-production-with-k8s.md

```diff
@@ -0,0 +1,383 @@
+---
+layout: page
+title: "Operations Guide to deploy Portworx in Production in DC/OS Clusters"
+keywords: operations guide, run book, disaster recovery, disaster proof, site failure, node failure, power failure
```

Comment:
The document name is "going-production-with-k8s.md". But the title is "Operations Guide to deploy Portworx in Production in DC/OS Clusters".

Please fix this.

Reply:
This is a WIP document and is on a branch. I will ask for reviews when I am done